app/controllers/concerns/find_common.rb

```diff
@@ -11,7 +11,7 @@ def find_resource
   def resource_finder(scope, id)
     raise ActiveRecord::RecordNotFound if scope.empty?
     result = scope.from_param(id) if scope.respond_to?(:from_param)
-    result ||= scope.friendly.find(id) if scope.respond_to?(:friendly)
+    result ||= scope.friendly.find(id) rescue nil if scope.respond_to?(:friendly)
```

Comment:
The point is that the inline rescue is also hiding errors that, 'scope', `scope.friendly` and `scope.friendly.find` could have, but the only one we're certain is legitimate is ResourceNotFound. Say 'scope.friendly' throws some other error because an update on friendly_id breaks it, we wouldn't know so easily as it falls back no matter what.

https://www.new-bamboo.co.uk/blog/2015/09/23/dont-inline-rescue-in-ruby/